packages/frontend/src/pages/ModelPlayground.svelte

```diff
@@ -73,6 +74,15 @@ function displayQuery(query: QueryState) {
   }
 }
 
+function requestTimeUpdate(start: number) {
+  requestAnimationFrame(() => {
+    if (inProgress) {
+      elapsed = (performance.now() - start) / 1000;
+      requestAnimationFrame(() => requestTimeUpdate(start));
```

Comment:
should this row just be `requestTimeUpdate(start)` as when it recalls itself it uses the `requestAnimationFrame` in any case?